resources/language/resource.language.fr_fr/strings.po

```diff
@@ -210,27 +210,27 @@ msgstr "Médiathèque"
 
 msgctxt "#30051"
 msgid "Profile set for playback from the library"
-msgstr ""
+msgstr "Profil utilisé lors de la lecture depuis la médiathèque"
 
 msgctxt "#30052"
 msgid "Set for library playback"
-msgstr ""
+msgstr "Utiliser ce profil lors de la lecture depuis la médiathèque"
```

Comment:
I don't know if it's possible in your language
you can find a shorter formula?
This is a context menu (you can see it by open context menu on a NF profile item)

Reply:
I removed the part "this profile" since it is quite obvious as the user opened the context menu on a profile so it should not confuse too much the user.
Do you think it is short enough now?